code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -511,4 +511,32 @@
 	ammo_type = /obj/item/ammo_casing/cap
 	matter = list(DEFAULT_WALL_MATERIAL = 600)
 	max_ammo = 7
-	multiple_sprites = 1
\ No newline at end of file
+	multiple_sprites = 1
+
+
+/obj/item/ammo_magazine/ma12gclip
```

Comment:
This is not named correctly at all in accordance to the new naming convention of ammo types.

The correct path should be /c12g and not /ma12gclip